verification/curator-service/api/src/model/user.ts

```diff
@@ -1,19 +1,12 @@
 import mongoose from 'mongoose';
 
 const userSchema = new mongoose.Schema({
-    name: {
-        type: String,
-    },
+    name: String,
     email: {
         type: String,
         required: 'User must have an email',
     },
-    googleID: {
-        type: String,
-        // Note: we can relax that requirement if we start
-        // supporting more identity platforms.
-        required: 'User must be logged-in with Google',
-    },
+    googleID: String,
```

Comment:
don't you need a "token" here in this schema?

Reply:
Ah, thanks for the catch. I had initially persisted the token, but went back and thought better of it. We exchange it for an identity which we persist (the email addr), which lets us avoid persisting the token itself. I'll remove the dangling line in the auth controller.